setup.py

```diff
@@ -177,32 +177,34 @@ def build_extension(self, ext):
 env = os.environ.copy()
 # ... build WarpX libraries with CMake
 #   note: changed default for SHARED, MPI, TESTING and EXAMPLES
-WarpX_COMPUTE = env.pop('WarpX_COMPUTE', 'OMP')
-WarpX_MPI = env.pop('WarpX_MPI', 'OFF')
-WarpX_EB = env.pop('WarpX_EB', 'OFF')
-WarpX_OPENPMD = env.pop('WarpX_OPENPMD', 'OFF')
-WarpX_PRECISION = env.pop('WarpX_PRECISION', 'DOUBLE')
-WarpX_PSATD = env.pop('WarpX_PSATD', 'OFF')
-WarpX_QED = env.pop('WarpX_QED', 'ON')
-WarpX_QED_TABLE_GEN = env.pop('WarpX_QED_TABLE_GEN', 'OFF')
-WarpX_DIMS = env.pop('WarpX_DIMS', '2;3;RZ')
+#   note: we use all-uppercase variable names for environment control to be
+#         consistent across platforms (especially Windows)
+WarpX_COMPUTE = env.pop('WARPX_COMPUTE', 'OMP')
```

Comment:
Perhaps make the Python variables also all caps as a reminder that they are from environment variables.

Reply:
Ok, done.